### PR TITLE
Surgery Steps visible on Operating computer

### PR DESCRIPTION
**Pull request by RemieRichards**
- Surgery steps are now visible on the operating computer
- Operating computer popup window changed to the sexier black ones
- Remove some of my old useless comments

### CODE REVIEW

File: code/modules/surgery/surgery.dm [hunk 1]
```diff
@@ -29,6 +29,46 @@
 	src = null
 
 
+proc/steps2text(var/datum/surgery_step/Step)
+	var/txt = "You're unsure how to proceeed"
```
Comment (Atskadan) [inline code/modules/surgery/surgery.dm]: proceed

Comment (RemieRichards) [inline code/modules/surgery/surgery.dm]: Ha, wow, Oki doki.

**Comment by ghost** [comment]
Operating computers useful now? Roboticists have a lower chance of stabbing people in the face with scalpels? Merge he!

File: code/modules/surgery/surgery.dm [hunk 2]
```diff
@@ -29,6 +29,46 @@
 	src = null
 
 
+proc/steps2text(var/datum/surgery_step/Step)
+	var/txt = "You're unsure how to proceed"
+
+	switch(Step.type)
```
Comment (Giacom) [inline code/modules/surgery/surgery.dm:35]: You could've very easily added a new variable to the surgery_step which has all of this.

**Pull request merged**